internals/daemon/daemon_test.go

```diff
@@ -461,9 +468,11 @@ func (s *daemonSuite) TestLoggedInUserAccess(c *C) {
 }
 
 func (s *daemonSuite) TestSuperAccess(c *C) {
-	d := s.newDaemon(c)
+	d := s.newDaemon(c, func(o *Options) {
+		o.AdditionalAdminUIDs = []sys.UserID{0xbadd00d}
```

Comment:
What about instead of adding this `daemonOpt` scaffolding, we just always add `0xbadd00d` to the list when we initialize the daemon in tests? (Similar above.)